example-apps/github/test/authentication.test.js

```diff
@@ -51,3 +52,52 @@ describe('oauth2 app', () => {
     );
   });
 });
+
+describe('getAccessToken', () => {
+  let result;
+
+  beforeEach(async () => {
+    nock('https://github.com/login/oauth')
+      .post('/access_token')
+      .reply(200, { access_token: 'someAccessToken' });
+    result = await appTester(App.authentication.oauth2Config.getAccessToken);
+  });
+
+  it('returns the expected tokens', () => {
+    expect(result.access_token).toBe('someAccessToken');
+  });
+});
+
+describe('testAuth', () => {
+  let result;
+  const bundle = {
+    // In production, these will be generated by Zapier and set automatically
+    inputData: {
+      state: '4444',
+      redirect_uri: 'https://zapier.com/',
+    },
+    environment: {
+      CLIENT_ID: process.env.CLIENT_ID,
+      CLIENT_SECRET: process.env.CLIENT_SECRET,
+    },
+  };
+
+  beforeEach(async () => {
+    nock('https://api.github.com')
+      .get('/user')
+      .reply(200, {
+        json: {
+          login: 'myLogin',
+        },
+      });
+    result = await appTester(App.authentication.test, bundle);
```

Comment:
Nitpick: This `result` object seems to be part of the `it('returns the expected info')` test scope instead of the `describe` scope
```
    const result = await appTester(App.authentication.test, bundle);
    expect(result.data.json.login).toBe('myLogin');
```